packages/@react-aria/dnd/src/index.ts

```diff
@@ -19,7 +19,14 @@ export type {DragPreviewProps} from './DragPreview';
 export type {DragOptions, DragResult} from './useDrag';
 export type {DropOptions, DropResult} from './useDrop';
 export type {ClipboardProps, ClipboardResult} from './useClipboard';
-export type {DropTargetDelegate} from '@react-types/shared';
+export type {
+  DropTargetDelegate,
+  DragItem,
+  DropItem,
+  TextItem,
+  FileItem,
+  DirectoryItem
```

Comment:
Should we rename these types to e.g. `TextDropItem` so it's more clear they are related to dnd? Especially when we add to the monopackage.